crates/xline/src/revision_number.rs

```diff
@@ -2,27 +2,34 @@ use std::sync::atomic::{AtomicI64, Ordering};
 
 /// Revision number
 #[derive(Debug)]
-pub(crate) struct RevisionNumberGenerator(AtomicI64);
+pub(crate) struct RevisionNumberGenerator {
+    current: AtomicI64,
+}
 
 impl RevisionNumberGenerator {
     /// Create a new revision
     pub(crate) fn new(rev: i64) -> Self {
-        Self(AtomicI64::new(rev))
+        Self {
+            current: AtomicI64::new(rev),
+        }
     }
 
-    /// Get the revision number
+    /// Get the current revision number
     pub(crate) fn get(&self) -> i64 {
-        self.0.load(Ordering::Relaxed)
-    }
-
-    /// Get the next revision number
-    pub(crate) fn next(&self) -> i64 {
-        self.0.fetch_add(1, Ordering::Relaxed).wrapping_add(1)
+        self.current.load(Ordering::Acquire)
```

Comment:
Why do you modify the memory order from `Relaxed` to `Acquire`?

Reply:
It could be shared between threads, potentially lead to unpredictable behavior.

Reply:
Reverted